appeals/web/src/server/appeals/appeal-details/interested-party-comments/edit-ip-comment/controller.js

```diff
@@ -52,17 +55,25 @@ export async function renderCheckAddress(request, response) {
  * @param {import('@pins/express/types/express.js').RenderedResponse<any, any, Number>} response
  * */
 export async function postCheckPage(request, response) {
-	const { currentAppeal, currentComment } = request;
+	const { currentAppeal, currentComment, session } = request;
 
 	await updateAddress(
 		request.apiClient,
 		currentAppeal.appealId,
 		currentComment.represented?.id,
-		request.body
+		request.session.editIpComment
```

Comment:
Not an issue, but as you've already destructured `session` from the request, this could just be `session.editIPComment`

Reply:
Will fix forward in my currently overlapping bug ticket